src/components/blog/BlogOverview.jsx

```diff
@@ -1,5 +1,42 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { Container, Row } from "react-bootstrap";
+import SingleBlogCard from "./SingleBlogCard";
+import "./style.overview.scss";
+import BlogHeader from "./BlogHeader";
 
 export default function BlogOverview() {
-  return <div></div>;
+  const [allBlogs, setAllBlogs] = useState([]);
+  const [blogsImages, setBlogsImages] = useState([]);
+
+  // To all of our data from wordpress
+
+  const getAllBlogsAndImages = async () => {
+    const blogs = await fetch("http://eecommerce.local/wp-json/wp/v2/health");
+    const blogsMedia = await fetch(
+      "http://eecommerce.local/wp-json/wp/v2/media"
+    );
+
```

Comment:
Can you please use `Promise.all` or any other practical simultanious fetch to make it faster? If I recall right, you were saying that fetching is slow, it could be due to this.

Reply:
+1 you should not be synchronously awaiting back-to-back on two fetches that don't depend on each other like this. use promise.all or you can move the awaits into line 21 and 22